content/doc/book/pipeline/development.adoc

```diff
@@ -52,9 +45,9 @@ Jenkins for secure command-line access.
 [source,bash]
 ----
 # ssh (Jenkins CLI)
-# JENKINS_SSHD_PORT=[sshd port on controller]
```

Comment:
This seems unrelated. Was it intentional?

Reply:
Yes, it was intentional so that the text layout on the page would not place a horizontal scroll bar on the example.

The example is close enough to the top of the page that it can collide with the navigation links on the right hand side of the page.  By reducing the text width, it reduces the chance of overlap that would add a horizontal scroll bar.

Reply:
https://deploy-preview-5307--jenkins-io-site-pr.netlify.app/doc/book/pipeline/development/ shows the result with this change.

![example-collision-zone](https://user-images.githubusercontent.com/156685/180899396-6e90fc4d-1768-406a-a0c8-3d83136365ab.png)

Without the change, it looks like this:

![current-page](https://user-images.githubusercontent.com/156685/180900106-ac7e1479-c741-41cf-a347-9553ad5c8bc3.png)